### PR TITLE
JI773438 P2022 2121 Fix-Incorrect font

### PR DESCRIPTION
**Pull request by Reden4005**
Add Roboto font to our project.

### CODE REVIEW

**Review by aniasgit — APPROVED** [review]
it fix the problem for me

File: apps/site/src/index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Patronage Site</title>
     <base href="/" />
-
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto&display=swap" />
```
Comment (pawelwojdas) [inline apps/site/src/index.html:7]: I would add Helvetica font as well to be consistent with fonts from material UI

**Review by Adys23 — APPROVED** [review]
LGTM :)

**Comment by krysia1** [comment]
LGTM 👍

**Review by krysia1 — APPROVED** [review]

File: apps/site/src/index.html (hunk 1 above)
Comment (Reden4005) [reply apps/site/src/index.html:7]: Helvetica is not a free font.

Comment (pawelwojdas) [reply apps/site/src/index.html:7]: Oh, I didn't know

**Review by pawelwojdas — APPROVED** [review]

File: apps/site/src/index.html (hunk 1 above)
Comment (kacperChlon) [reply apps/site/src/index.html:7]: Please use self hosted fonts instead of linked that way. Please provide proper files with proper extensions fonts.

Comment (Reden4005) [reply apps/site/src/index.html]: Done

File: apps/site/src/main.tsx [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import "./i18n";
+import "./styles.css";
```
Comment (kacperChlon) [inline apps/site/src/main.tsx:2]: change the name to global-styles.css and move to shared directory

File: apps/site/src/styles.css [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/* roboto-italic - latin-ext */
```
Comment (kacperChlon) [inline apps/site/src/styles.css:1]: please move this file to assets or styles folder in shared directory

File: apps/site/src/styles.css [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+@font-face {
+  font-family: "Roboto";
+  font-style: italic;
+  font-weight: 400;
+  src: url("./fonts/roboto/roboto-italic.eot");
+  src: local(""), url("./fonts/roboto/roboto-italic.eot") format("embedded-opentype"),
```
Comment (kacperChlon) [inline apps/site/src/styles.css:7]: if we don't support local fonts then local() function is not necessary

File: apps/site/src/main.tsx (hunk 2 above)
Comment (Reden4005) [reply apps/site/src/main.tsx]: Refactored.

File: apps/site/src/styles.css (hunk 4 above)
Comment (Reden4005) [reply apps/site/src/styles.css]: ok

**Pull request merged**